src/main/java/org/isf/menu/gui/GroupEdit.java

```diff
@@ -155,7 +157,15 @@ private JPanel getDataPanel() {
 			  
 			dataPanel.add(descLabel);
 			dataPanel.add(getDescriptionTextField());
-			SpringUtilities.makeCompactGrid(dataPanel, 2, 2, 5, 5, 5, 5);
+
+			if (!insert && group.isDeleted()) {
```

Comment:
I would remove this `if` condition to show the checkbox in any case.
Please adjust the window size:

![image](https://github.com/user-attachments/assets/c201bf56-39ff-444c-bc8e-c4c054556dad)